### PR TITLE
Fixed workflow name issue, added info popover for private image registry

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amit@chaosnative.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx [hunk 1]
```diff
@@ -346,9 +368,43 @@ const ImageRegistry = () => {
...
+                          >
+                            <Typography className={classes.infoText}>
+                              {t('settings.imageRegistry.provideInfo')}{' '}
+                              <a
+                                href="https://kubernetes.io/docs/concepts/configuration/secret/"
+                                target="_"
+                              >
+                                {t('settings.imageRegistry.visit')}
+                              </a>
+                              .
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: is this needed?

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/styles.ts [hunk 2]
```diff
@@ -62,6 +62,14 @@ const useStyles = makeStyles((theme: Theme) => ({
   additionalDetails: {
     fontSize: '1rem',
     marginTop: theme.spacing(2.5),
+    marginRight: theme.spacing(1),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Settings/ImageRegistry/styles.ts:65]: add just one margin property

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: The document url?

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/styles.ts (hunk 2 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/styles.ts:65]: Sure

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: The '.' @amityt ?

**Review by S-ayanide — APPROVED** [review]

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: Ohh, that full stop wont cause any issue @ishangupta-ds

Comment (S-ayanide) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: I think this is a full-stop at the end of the tooltip

**Review by ishangupta-ds — APPROVED** [review]

File: litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: Attaching screenshot for reference
Screenshot from 2021-05-14 16-10-17

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/Settings/ImageRegistry/index.tsx:404]: Could have been in translation itself @amityt

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**